Packet++/header/IPv4Layer.h

```diff
@@ -280,7 +280,7 @@ namespace pcpp
 
 			res.type = (IPv4TimestampOptionValue::TimestampType)m_Data->recordValue[1];
 
-			uint8_t valueOffset = (uint8_t)(2);
+			uint8_t valueOffset = 2;
```

Comment:
I think removing the casting might cause a compilation warning on some platfroms